.travis.yml

```diff
@@ -20,11 +20,9 @@ notifications:
   email: false
 
 # script always run to completion (set +e).
-install:
-  - go get -t -v $(go list ./ ... | grep -v examples)
```

Comment:
I missed this the first time I reviewed. I happened to inherit most of this project. Would you explain what this line did and why we can remove it?  Thank you.

Reply:
It's no longer needed because all the `go` tooling is module-aware now, and automatically fetches dependencies. IIRC it was causing an error with the version of Go I was using, so I took it out.